server/utils/helpers/commentCtrlHelper.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Format and return the retrieved comments from the DB
+ * @param {Array} rawComments array of unformatted comments
+ * @returns {Array} an array of comments
+ */
+const parseComments = (rawComments) => {
+  if (rawComments[0].id === null) return [];
+  return rawComments.map(elem => ({
```

Comment:
Is there no alternative to using loops here. What if you have a large amounts of comments for the article. I think this would reduce the speed of the server